src/sass/_style.scss

```diff
@@ -512,6 +512,10 @@ select.form-control {
   }
 }
 
+.content {
```

Comment:
Could you please point me to this class's usage?

Reply:
This was related to neighboring PR about horizontal scroll and no valid for now. I removed this style